src/System.Runtime.Serialization.Primitives/src/Configurations.props

```diff
@@ -4,7 +4,7 @@
     <BuildConfigurations>
       netcore50aot-Windows_NT;
       uap101aot-Windows_NT;
-      net463-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
Because it was specified before.  The only significant difference for netfx is that the OSGroup will be set to Windows_NT, as well as TargetsWindows will be set to true.  If these properties are used by the project the it should set Windows_NT.